src/app/view/paperView.js

```diff
@@ -19,6 +19,7 @@ import * as DXFSolidObjectRenderer from "./render2D/dxfSolidObjectRenderer2D";
 import Layer from "../core/layer";
 import Device from "../core/device";
 import Feature from "../core/feature";
+import components from "@dagrejs/graphlib/lib/alg/components";
```

Comment:
I don't think this reference should be used here

Reply:
Is this the task: Delete 3DuF.js from the source tree?